### PR TITLE
CI: remove inputs from docker_server job command

### PR DESCRIPTION
**Pull request by maxknv**
#job_style_check #job_Docker_server_image #job_Docker_keeper_image

### Changelog category (leave one):
- CI Fix or Improvement (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
eliminates the need to provide input args to docker server jobs to clean yml files

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by maxknv** [comment]
Further efforts to unblock https://github.com/ClickHouse/ClickHouse/pull/60421

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 2123a8ca352ba4f131329d0aa225549cf8d90c84 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/60602/2123a8ca352ba4f131329d0aa225549cf8d90c84/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: tests/ci/docker_server.py [hunk 1]
```diff
@@ -337,13 +331,37 @@ def main():
     makedirs(TEMP_PATH, exist_ok=True)
 
     args = parse_args()
-    image = DockerImageData(args.image_path, args.image_repo, False)
+
+    pr_info = PRInfo()
+    check_name = getenv("CHECK_NAME", "")
```
Comment (Felixoid) [inline tests/ci/docker_server.py:336]: I'd expect it in `env_helper`, then we'll just use `CHECK_NAME` var

File: tests/ci/docker_server.py [hunk 2]
```diff
@@ -355,13 +373,13 @@ def main():
             repo_urls[arch] = f"{args.bucket_prefix}/{build_name}"
         if args.allow_build_reuse:
             # read s3 urls from pre-downloaded build reports
-            if "clickhouse-server" in args.image_repo:
+            if "clickhouse-server" in image_repo:
```
Comment (Felixoid) [inline tests/ci/docker_server.py:376]: we control now `image_repo`, and we can code it into some class even. With PACKAGES together

**Review by Felixoid — COMMENTED** [review]
I'd say, it's pretty solid. We can improve it a little bit by juggling parameters

File: tests/ci/docker_server.py (hunk 2 above)
Comment (maxknv) [reply tests/ci/docker_server.py:379]: I'll leave creating a class for later

**Pull request merged**